ansible/host_vars/linux-03.ih-eu-mda1.nimbus.mainnet.yml

```diff
@@ -2,6 +2,11 @@
 era_files_domain: 'mainnet.era.nimbus.team'
 era_files_path: '/data/era'
 
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```

Comment:
Please remove this. Look at `group_vars`.

Reply:
done.